crates/sdk/src/types.rs

```diff
@@ -0,0 +1,87 @@
+use itertools::Itertools;
+use openvm_native_recursion::halo2::{Fr, RawEvmProof};
+use serde::{Deserialize, Serialize};
+use serde_with::serde_as;
+
+const BN254_BYTES: usize = 32;
+const NUM_BN254_ACCUMULATORS: usize = 12;
+const NUM_BN254_PROOF: usize = 43;
+
+#[serde_as]
+#[derive(Clone, Deserialize, Serialize)]
+pub struct EvmProof {
+    #[serde_as(as = "serde_with::hex::Hex")]
+    pub accumulators: Vec<u8>,
```

Comment:
can you document the endian-ness of different fields? I remember there was something about instances being big but proof being little